examples/numerical/examples_helpers.py

```diff
@@ -129,3 +129,11 @@ def compute_inverse_denominators(H: dict, nocc: list[int], nvir: list[int], rank
             f"compute_inverse_denominators() supports rank up to 3, but was called with rank = {rank}"
         )
     return D
+
```

Comment:
Could you please add a comment in the style of the function `update_cc_amplitudes`?

Reply:
Done!